dcplib/aws/dynamodb.py

```diff
@@ -0,0 +1,134 @@
+from typing import Generator, Optional
+
```

Comment:
rathern then passing in the client, just import a dynamoDB client.
```suggestion
from .clients import dynamodb as db
```

Reply:
Won't this use the dcplib client rather than the dss client then?